src/decision/Criteria.jl

```diff
@@ -44,6 +44,15 @@ Base.@kwdef struct Criteria{P,N} <: EcoModel
         name="Outgoing Connectivity (Seed)",
         description="Higher values give preference to locations with high outgoing connectivity (i.e., provides larvae to other sites) for enhanced coral deployments.",
     )
+    seed_depth::P = Param(
+        1.0;
+        ptype="real",
+        bounds=(0.0, 1.0),
+        default_bounds=(0.0, 1.0),
+        dists="unif",
+        name="Depth (Seed)",
+        description="Higher values give preference to depper locations for enhanced coral deployments.",
```

Comment:
Here you are describing how you intend to use, but I think it should be a description of what this is. Something like "The depth of a location". Because we can use for other things later on, right?

Reply:
This is not the depth of a location, it is the weighting for the depth of a location criteria in the location selection (scalar). It is only used in deciding where to seed.